src/app.py

```diff
@@ -18,4 +18,4 @@ async def lifespan(app: FastAPI):
 app = FastAPI(lifespan=lifespan)
 app.add_middleware(GZipMiddleware, minimum_size=100)
 app.include_router(scene_router.router)
-app.include_router(server_router.router)
```

Comment:
Please rollback this change. Without it, diff will think that adding anything to the end of this file changes `app.include_router(server_router.router)`